src/utils/raw.rs

```diff
@@ -214,31 +210,11 @@ fn symlink_junction_inner(target: &Path, junction: &Path) -> io::Result<()> {
     }
 }
 
-pub fn hardlink(src: &Path, dest: &Path) -> io::Result<()> {
+pub(crate) fn hardlink(src: &Path, dest: &Path) -> io::Result<()> {
     let _ = fs::remove_file(dest);
     fs::hard_link(src, dest)
 }
 
-#[derive(Debug, ThisError)]
-pub enum CommandError {
```

Comment:
So this is just ununused?

Reply:
Yes. It doesn't look like it's still being used anywhere else.